### PR TITLE
Postgres & Cassandra Setup Complete

### PR DESCRIPTION
**Pull request by twzkraus**
- `generate-records.js` writes files for convenient insert into both Cassandra and Postgres
- `generate-and-insert.js` includes instructions for seeding databases
- `postgres.js` prepares Sequelize schema
- Few updates to `app.js` which weren't already reflected in master

### CODE REVIEW

File: database/db-router.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  },
+  getOne: (id) => {
+    return new Promise((res, rej) => res(MongoItem.find({ id: parseInt(id) })));
+  },
+  getAll: (id) => {
+    return new Promise((res, rej) => res(MongoItem.find({})));
+  },
+};
+
+const postgres = {
```
Comment (ardydj) [inline database/db-router.js:33]: This is really cool and useful!

File: database/generate-records.js [hunk 2]
```diff
@@ -63,34 +91,52 @@ const writeNRecords = (n, document, callback) => {
...
+    if (i > 0 && !ok2) {
+      noSQLSheet.once('drain', write);
+    }
   }
   write();
 };
 
 module.exports = (n, callback) => {
-  writeNRecords(n, writeRecords, callback);
+  writeImageRecords(imageDataSheet);
```
Comment (ardydj) [inline database/generate-records.js:140]: What is the difference between throughSheet and noSQLSheet?

**Review by ardydj — APPROVED** [review]
Looks good! My one question is on the differentiation between throughSheet and noSQLsheet since currently I don't fully understand that implementation.

File: database/generate-records.js (hunk 2 above)
Comment (twzkraus) [reply database/generate-records.js:140]: yeah, to be honest my naming convention is pretty bad here. throughSheet is one of three tables for PostgreSQL--it has 2 columns to relate image ids to product ids. noSQLSheet is the one and only table for Cassandra. It practically stores the entire object that I want returned on the client side.

**Pull request merged**